frontend/app/routes/_public+/apply+/$id+/date-of-birth.tsx

```diff
@@ -102,13 +102,13 @@ export default function ApplyFlowDateOfBirth() {
           <InputField id="year" label={t('date-of-birth.year')} name="year" type="number" min={1900} errorMessage={errorMessages.year} defaultValue={state?.year} />
         </div>
         <div className="mt-6 flex flex-wrap items-center gap-3">
-          <ButtonLink type="button" variant="alternative" to={`/apply/${id}/tax-filing`}>
+          <ButtonLink id="back-button" to={`/apply`}>
```

Comment:
No need for JSX curly braces unless the url path is not good and requires `id`.

```suggestion
          <ButtonLink id="back-button" to="/apply">
```